scripts/config/lxdialog/dialog.h

```diff
@@ -157,7 +157,7 @@ struct dialog_item {
 	int selected;	/* Set to 1 by dialog_*() function if selected. */
 };
 
-/* list of lialog_items */
+/* list of dialog_items */
```

Comment:
It's possible that this is lxdialog, but it seemed unlikely based on the preceding block